src/zappy_ai_src/zappy_ai

```diff
@@ -22,6 +22,19 @@ def needHelp(ac : int, av) ->  bool:
             return True
     return False
 
+def getIPFromHostname(hostname : str) -> str:
+    """ Find the IP corresponding to the Hostname, raise an RunTimeError if there isn't """
+    Ip = None
+    with open("/etc/hosts") as file:
```

Comment:
no

Reply:
You should use functions that does the resolving for you. Not doing it by hand

Reply:
Can u explain savon pls